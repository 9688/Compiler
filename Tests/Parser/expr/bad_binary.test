4 !=||;
a + ;
a *, x;