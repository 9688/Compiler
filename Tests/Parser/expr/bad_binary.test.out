└──''
Ошибка синтаксиса в строке 1 позиции 5: требуется выражение

Ошибка синтаксиса в строке 2 позиции 1: требуется ";"

Ошибка синтаксиса в строке 3 позиции 4: требуется выражение

Ошибка синтаксиса в строке 3 позиции 8: требуется ";"


